pkg/controller/controller.go

```diff
@@ -282,6 +282,7 @@ func (c *Controller) runController() {
 
 		case <-runTimer.After(interval):
```

Comment:
Is it on purpose that we do nothing after the timer elapses? Go doesn't do case fallthrough.

Reply:
Indentation seems to be a bit off as well :-)

Reply:
> Is it on purpose that we do nothing after the timer elapses? Go doesn't do case fallthrough.

We do "something". Once that timer elapses we run the function in 
https://github.com/cilium/cilium/blob/a8e3fa283cf40b330035265bf83d63800b296bcf/pkg/controller/controller.go#L206

> Indentation seems to be a bit off as well :-)

The indentation is correct, otherwise gofmt would have complained.

Reply:
Ah now I get the logic. And I guess it's just github not showing it properly.